packages/react/src/Screen/Screen.tsx

```diff
@@ -6,7 +6,7 @@
 import clsx from 'clsx'
 import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
 
-type ScreenMaxWidth = 'wide' | 'extra-wide'
+type ScreenMaxWidth = 'wide' | 'x-wide'
```

Comment:
Do we want to change the interface for Screen as well? We'll probably never need a `2x-wide` Screen, right?

Reply:
Maybe, maybe not – but either way, why maintain two approaches?

Reply:
True, consistency might be useful here.

Just had another brain fart: shouldn't this be `medium` and `wide`? 100rem is our default / main width, right? In most cases we use `medium` for the default size.

Reply:
Yes we do try to use `medium` for the default. Our set of breakpoints is why we haven’t succeeded for `Screen` (just yet):

- 0px – 575px = narrow
- 576px – 1087px = medium
- 1088px – 1599px = wide
- 1600px – 2112px = extra wide

So here, `100rem` is the maximum width for ‘wide’ screens and `132rem` the max for ‘extra wide’.

When reasoning about ‘desktop designs’, you’d expect a viewport width in the region of 1280, 1440, 1536 pixels. A good reason to call them ‘medium’. But we have named them ‘wide’ screens (based on the spreadsheet we designed the grid values with).

Naming them the ‘default’ sounds attractive, but then ‘tablet’ windows must become ‘narrow’ and ‘phone’ windows ‘extra narrow’. Would you prefer that?

It will add a bit of complexity to everyday speech – I think we chose the current approach so that we can refer to phones as having ‘narrow screens’ rather than ‘extra narrow’ ones – but I do agree this would be more precise and very much defensible.

Reply:
Hm, I'm not sure the props for Screen should be related to the grid and our breakpoints. Screen is just a container which sets a maximum width, and that maximum width is the default, or a wider variant.

Having said that: maybe this isn't the PR to discuss this, if you feel the prop has the right options, we can leave it for now and merge this PR

Reply:
Fair point. At the same time I think the pixel widths of the breakpoints and max widths are related. I’ve made a note to discuss.